docs/release.md

```diff
@@ -2,6 +2,22 @@
 
 We use [Changesets](https://github.com/changesets/changesets) to version and release our packages.
 
+When we merge to main, the release workflow will:
+
+1. Create a release pull request with:
+   - A version bump for the package calculated by the changesets.
+   - A changelog entry summarizing the changes in the release.
+1. Create an `alpha` version of the package on whatever is merged to main, and you can install it with `npm install @browserbasehq/stagehand@alpha`. This is useful for testing the release before it's published to the `latest` tag.
+
+When the pull request is merged, the release workflow will publish the package to npm with the version calculated by the changesets.
+
+For more information on how changesets work, see the [changesets docs](https://github.com/changesets/changesets) and our [release.yml file](/.github/workflows/release.yml).
+
+# Manually Releasing
+
+> **Warning**
```

Comment:
this will render nicer on github

```suggestion
> [!WARNING]
```

Reply:
786b6974afb339d4dbbd63ba3acd27f87c00d5ab